packages/prettier-config/README.md

```diff
@@ -0,0 +1,17 @@
+# Prettier
+
+Provide reusable prettier config
+
+1. Install the peerDependencies of prettier config:
+
+```sh
+npx install-peerdeps -Y --dev @offchainlabs/prettier-config
+```
+
+2. Usage
+
+```ts
```

Comment:
nit: maybe be explicit that this should be done in a Prettier config file, e.g. `prettier.config.js`

Reply:
Update with the different possibilities